arci/src/clients/joint_trajectory_clients_container.rs

```diff
@@ -1,7 +1,6 @@
 use crate::error::Error;
 use crate::traits::{JointTrajectoryClient, TrajectoryPoint};
-use crate::waits::WaitFuture;
-use futures::stream::FuturesOrdered;
+use async_trait::async_trait;
 
 pub struct JointTrajectoryClientsContainer<T: JointTrajectoryClient> {
```

Comment:
Please do not change except for your test.

Reply:
I fixed it.
Sorry data was disturbed, these codes are old one.